BLSQ/iaso
# IA-3280: Completeness stats: period filter is broken

Explain what problem this PR is resolving
- Completeness stats: period filter is broken
- This PR is about to display a message which indicates that the selected form has no period type
Related JIRA tickets : [IA-3280](https://bluesquare.atlassian.net/browse/IA-3280)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- Display in the periodPicker a different message when the selected form has no period type vs when there is no form selected

## How to test
- Create or use existing form which has no period as period type
- Go to completeness stats and select a form with no period type and select another one with a period type
- Check at each test the period picker behavior

## Print screen / video
[Screencast from 2024-08-08 14-52-40.webm](https://github.com/user-attachments/assets/6635eb86-cfbc-4abd-94af-83856205d843)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3280]: https://bluesquare.atlassian.net/browse/IA-3280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
